tezos-merkle/changelog.md

```diff
@@ -44,7 +44,3 @@ This project adheres to
   - Fast element appends
   - Leaf-to-root paths generator
 
-[0.1.0]: https://gitlab.com/tzstamp/tezos-merkle/-/releases/0.1.0
-[0.2.0]: https://gitlab.com/tzstamp/tezos-merkle/-/releases/0.2.0
-[0.3.0]: https://gitlab.com/tzstamp/tezos-merkle/-/releases/0.3.0
-[0.3.1]: https://gitlab.com/tzstamp/tezos-merkle/-/releases/0.3.1
```

Comment:
we can remake this releases and point to our github

Reply:
I re-added previous GitLab releases and added the new `0.3.2` release.